src/motor_instructions.py

```diff
@@ -2,14 +2,8 @@
 
 import rospy
 from std_msgs.msg import Empty, Float32
-import imp
-import rospkg
-
-rospack = rospkg.RosPack()
-pck_path = rospack.get_path("bike_sentry")
-
-SystemState = imp.load_source("bike_sentry.SystemState", pck_path + "/scripts/SystemState.py")
-CDM = imp.load_source("module.name", pck_path + "/scripts/CameraDataManager.py")
+from CameraDataManager import CameraDataManager
```

Comment:
Nice, I assume you made CameraDataManager into a package? Was it hard?

Reply:
Just to simplify everything I moved it to the same directory as our other code. Python will find it if its the same directory.